vassal-app/src/main/java/VASSAL/counters/FreeRotator.java

```diff
@@ -981,12 +1001,13 @@ public String getType() {
         .append(rndCommand.getValueString() == null ? "" : rndCommand.getValueString().trim());
       se.append(nameConfig.getValueString());
       se.append(descConfig.getValueString());
+      se.append(startAngleConfig.getValueString());
       return ID + se.getValue();
     }
 
     @Override
     public String getState() {
-      return "0";
+      return startAngleConfig.getValueString();
```

Comment:
return rotator.getState();

Reply:
Ah, I know why they returned "0". For my solution, you would need to check if the type has changed from fixed to free or v/v and convert the starting state if it is non-zero.